File: market/test/java/ru/yandex/market/wms/packing/config/PickByLightTestConfig.java
package ru.yandex.market.wms.packing.config;

import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Import;

import ru.yandex.market.wms.pickbylight.client.mock.PickByLightMockClient;

@Configuration
@Import(PickByLightMockClient.class)
public class PickByLightTestConfig {
}

File: market/test/kotlin/ru/yandex/market/wms/consolidation/config/PickByLightTestConfig.kt
package ru.yandex.market.wms.consolidation.config

import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Import
import ru.yandex.market.wms.pickbylight.client.mock.PickByLightMockClient

@Configuration
@Import(PickByLightMockClient::class)
class PickByLightTestConfig
